doc/WindowsTestPasses.md

```diff
@@ -26,10 +28,10 @@ Prerequisites:
 
 1. Open Visual Studio 2017 and use the TDP drop-down menu to open the `Device Manager`.
 1. In the pane that opens to the left, choose `Add` and then `Nebula VM Device`. Nebula is a cloud provider for VMs (like Azure but a more private instance for corporate work usage).
-1. Name the machine and choose the build/branch/flavor/SKU from the drop downs at the bottom. It will find the VHD for you from the build shares. Hit `Add Device` to deploy to Nebula.
+1. Name the machine and choose the build/branch/flavor/SKU rom the drop downs at the bottom. It will find the VHD for you from the build shares. Hit `Add Device` to deploy to Nebula.

```

Comment:
This change confuses me.

Reply:
Oh sorry. I must have accidentally deleted a character. I'll fix it.